.github/pull_request_template.md

```diff
@@ -2,6 +2,8 @@
 <!---
 Describe your changes, and why you're making them. Is this linked to an open
 issue, a pull request on dbt core, etc?
+
+To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](/contributing/content-style-guide). 
```

Comment:
I think we need to use the absolute link to the `current` branch. We can test once Matt merges his PR
```suggestion
To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/contributing/content-style-guide.md). 
```